pkg/parser_test.go

```diff
@@ -0,0 +1,12 @@
+package pkg
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewParser(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(nil)
```

Comment:
This is just a dummy Test func. I havent written any tests for parser. Need to remove it. This has totally slipped my attentions, since, well, since it doesnt return any error on ```go test``` :-D